syntax = "proto3";

//// https://protobuf.dev/reference/cpp/cpp-generated/#message
//option optimize_for = CODE_SIZE;

package Odb.Lib.Protobuf;

//
//  StepDirectory.h
//

message AttrListFile {

    optional string directory = 1;
    optional string path = 2;
    optional string units = 3;
    map<string, string> attributesByName = 4;
}